lib/logger.js

```diff
@@ -1,27 +1,46 @@
 'use strict';
 
 const Winston = require('winston');
+const morgan = require('morgan');
+
+function Logger(level, filename) {
+  const logLevel = level || 'info';
 
-function Logger(config) {
   const logger = new Winston.Logger({
-    level: config.get('log:level'),
+    level: logLevel,
     transports: [
       new Winston.transports.Console({
         colorize: true,
         timestamp: true
       })
-    ]
+    ],
+    exitOnError: false
```

Comment:
This only applies to caught unhanded-exceptions, which the logger should not be listening for. In any case, the process _should_ crash in the event of an unhanded-exception allowing Upstart to restart it.